pkg/apis/proto/mapstream/v1/mapstream.proto

```diff
@@ -17,11 +17,11 @@ limitations under the License.
 syntax = "proto3";
 
 option go_package = "github.com/numaproj/numaflow-go/pkg/apis/proto/mapstream/v1";
-option java_package = "io.numaproj.numaflow.mapstream.v1";
```

Comment:
Keep all of these `java_packages`, eventually we will use them as the source of truth.